IDE/IAR/wolfBoot-test-app.eww

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
When I open this project with IAR EWARM 8.30.1 I get the error `Non-recognized options were detected in the project file`.

![Screen Shot 2020-04-07 at 7 26 44 AM](https://user-images.githubusercontent.com/2692870/78683113-bcc12b80-78a3-11ea-937f-a75b6538e5a9.png)

I pushed an update here: https://github.com/dgarske/wolfBoot/commit/2a2ead91eea0f507b5b0299b78cc75c50ba7b3a3

Reply:
Seems a conflict between IAR v. 8.30.1 and 8.50.1 - I have the same problem if I open your project, suggest to leave it bound to latest version at the time of this PR